packages/angular_devkit/build_angular/src/angular-cli-files/models/webpack-configs/browser.ts

```diff
@@ -67,9 +67,9 @@ export function getBrowserConfig(wco: WebpackConfigOptions) {
 
   if (!isEval && (scriptsSourceMap || stylesSourceMap)) {
     extraPlugins.push(getSourceMapDevTool(
-      scriptsSourceMap,
-      stylesSourceMap,
-      hiddenSourceMap,
+      scriptsSourceMap === undefined ? true : scriptsSourceMap,
```

Comment:
Can we use the normalizeSourceMapOptions method for this cases, setting these default in multiple places seems risky.

Reply:
We should, or at least add MORE normalizing methods.